pkgs/development/tools/misc/nxpmicro-mfgtools/default.nix

```diff
@@ -25,6 +25,12 @@ stdenv.mkDerivation rec {
 
   preConfigure = "echo ${version} > .tarball-version";
 
+  postInstall = ''
+    $out/bin/uuu -udev > udev-rules 2>stderr.txt # rules printed by this invocation are static, they come from hardcoded configs in libuuu/config.cpp:48
```

Comment:
```suggestion
    # rules printed by this invocation are static, they come from hardcoded configs in libuuu/config.cpp:48
    $out/bin/uuu -udev > udev-rules 2>stderr.txt
```

Reply:
Moved, and also split the comment into 2 lines